plugins/examples/resource-charts/src/index.tsx

```diff
@@ -0,0 +1,52 @@
+import { K8s, registerOverviewChartsProcessor } from '@kinvolk/headlamp-plugin/lib';
+import { TileChart } from '@kinvolk/headlamp-plugin/lib/CommonComponents';
+import { Box, Paper } from '@mui/material';
+import { useTheme } from '@mui/material/styles';
+import React from 'react';
+
```

Comment:
I think a short comment here explaining what this plugin is showing would be good. Something like: 

> This shows how to add a custom overview chart on the overview page.